ts/features/fci/components/SuccessComponent.tsx

```diff
@@ -70,6 +73,7 @@ const SuccessComponent = (props: {
   switch (status) {
     case SignatureRequestStatusEnum.WAIT_FOR_SIGNATURE:
       dispatch(fciStartRequest());
+      trackFciDocOpening(expires_at, fciDocuments.length);
```

Comment:
In every other cases you did `track` first and then `dispatch`. What's the best way to do this? I personally prefer tracking first and then dispatching. Anyway, I suggest to use always the same pattern. What do you think?

Reply:
Yep. It's a good point. Make sense to track first. Thx!